### DISCUSSION TITLE
Staging

### DISCUSSION THREAD

**Discussion by dhakelila** [Vizzuality/mangrove-atlas#302] [General]
HI Lammert and Kate, [staging](http://staging.globalmangrovewatch.org/) is been updated with the latest changes. More will come along the day and hopefully, everything will be done today. 

Cheers!

**Comment by dhakelila** [comment]
Hi @lhilarides , @klongleywood 

Here is the current status of staging: 

**Modify platform such that widgets and datasets are arranged by category**
Done. Missing "all" category. 

**Remove Highlighted Places widget**
Done. 

**Refine default scene so that mangrove extent is more visible**
Done. Please, let us know if it works for you. Personally, I like very much the result. 

**Allow users to download selected spatial data (which we own)**
Done. 

Missing: 
- Capacity to hide the widgets and see only the map. I'm not sure we'll have the time to implement this on time. 
- Blue carbon raster export: Please, check [Tamara's messages about this](https://github.com/Vizzuality/mangrove-atlas/discussions/295). 
- There is a bug with the button to collapse/expand all the widgets that we are looking into. 


Please, share your thoughts.

**Reply by klongleywood** [reply]
Thanks! I like the changes to the default scene.  Should the staging site be letting us jump to other locations other than worldwide? It crashes when I do that, but maybe it's not intended to have that feature. Just wanted to flag in case that would be an issue for launch.

**Reply by klongleywood** [reply]
Hi again -- what happened to the mangrove biomass and height widgets?

**Reply by dhakelila** [reply]
Good morning Kate! 

The site is supposed to navigate to different locations and it's working correctly for us. Both by searching and by clicking on the map. Does this issue persist for you? Could you clear your cache and  try again? 

<img width="1072" alt="biomass and hieght for AUS" src="https://user-images.githubusercontent.com/6608294/126942440-6d8a3840-271e-4faf-807d-fcb17bcb701d.png">


<img width="554" alt="categories" src="https://user-images.githubusercontent.com/6608294/126941996-a245bda9-280d-4014-90ea-522c8601d692.png">

Regarding the "height" and "biomass" widgets, those weren't included in the categorization we got, therefore, you can only see those when going to the "all" category. Let us know if those should belong to one of the proposed categories. Maybe the plan was at some point to remove them and that is why they are not on the categorization? Remember those widgets are only available for locations different than Worldwide. 

<img width="1072" alt="biomass and hieght for AUS" src="https://user-images.githubusercontent.com/6608294/126942440-6d8a3840-271e-4faf-807d-fcb17bcb701d.png">



https://user-images.githubusercontent.com/6608294/126943956-84a684de-8926-427a-a4f3-b5e8ae474e92.mov

**Reply by klongleywood** [reply]
This seems to be working on my end now, and everything else seems to be running smoothly as well. 

I just have a few comments/questions about the downloads and then I think we can be ready to go: 

1) When I go to try to download a dataset, it makes me sign in with a Google account. This seems like kind of an odd set up to start, so I'd like to see that be more seamless if possible, but then once I do sign in, I'm getting an error message: No such object: mangrove_atlas/land-cover/gmw2011v2_0.tif

2. Mangrove net change looks like it points users back to the extent -- didn't we have change layers available for download too?

3. Under mangrove blue carbon -- since there is no download available yet for the aboveground carbon, should we not have it on the menu? For the soil carbon, can't we just point them here? https://zenodo.org/record/2536803#.YP6xOuhKg2x

4. For mangrove height, the download link can point them to the same site as mangrove biomass. https://daac.ornl.gov/CMS/guides/CMS_Global_Map_Mangrove_Canopy.html

**Reply by dhakelila** [reply]
Hi Kate, we are reviewing the download. Some answers: 

1. This is weird. It works well on my end. The rest of the team is also checking. But could you retry? 
2. Indeed we are pointing to the same files, but it is what was marked on the excel file we got. 
![Screenshot 2021-07-26 at 15 18 39](https://user-images.githubusercontent.com/6608294/126995313-dc10897c-1486-4182-806a-aa950f79162e.png)
3. Ok. 
4. Ok.

**Comment by klongleywood** [comment]
It seems like this is only happening when I'm in Chrome incognito mode -- it works fine in the regular version of Chrome, but that's still odd...likely not a huge problem. 

As for the changes - yes, I'm seeing now that's what we provided. Let's keep as is for now unless @lhilarides meant to point elsewhere for these?

**Reply by dhakelila** [reply]
Ok. The developers are taking care of the rest of the items. 

Let me check in incognito mode. It is indeed weird.

**Reply by dhakelila** [reply]
It also requires me to log-in to download the files. But then, it downloads correctly. I'm wondering if there is a way to skip this step. Since we are downloading directly from google cloud, it might be required to sing-in... 
I'm checking with the team. 

I'll keep you posted on this but I assume this can be reviewed after the release.

**Reply by dhakelila** [reply]
Kate, the developers found the issue with points 1 and 2. The URL was wrong. It should point at  `https://storage.googleapis.com/mangrove_atlas` instead of ` https://storage.cloud.google.com/mangrove-atlas`

**Comment by dhakelila** [comment]
@klongleywood, we are fixing the download issues. 

Shall we deploy to production once those are done? The developers are still on it. But I just want to speed things up.

**Reply by klongleywood** [reply]
Yes, go for it!

**Reply by dhakelila** [reply]
Hi Kate, our day its coming to its end and I would prefer to deploy tomorrow morning and review it slowly. Is there any chance that we can deploy tomorrow?

**Comment by klongleywood** [comment]
Hi Clara,

That’s fine with me – the mangrove report is making a big splash today and I think it’s fine if we announce the changes tomorrow.

  Kate

**Reply by dhakelila** [reply]
Thank you so much for your comprehension! **We'll deploy to production tomorrow our morning** so I hope that everything will be fine by the time you wake up! 

Thanks!  See you tomorrow.

**Comment by lyubovpencheva** [comment]
this is done.
